scripts/merge-and-autograde.sh

```diff
@@ -2,7 +2,8 @@
 
 # usage
 display_usage() {
```

Comment:
Nice work. A good change here might be to rename the method display_usage() to something like display_usage_and_exit() since its internal working has now changed.

Reply:
Yep makes sense, I'll make the changes.